collector.go

```diff
@@ -602,6 +602,7 @@ func indexesToLabels(indexOids []int, metric *config.Metric, oidToPdu map[string
 		}
 		if pdu, ok := oidToPdu[oid]; ok {
 			labels[lookup.Labelname] = pduValueAsString(&pdu, lookup.Type)
+			labelOids[lookup.Labelname] = oidToList(labels[lookup.Labelname])
```

Comment:
You're presuming that the value is an oid, which isn't generally true.

Reply:
That is true but I cannot see any harm doing so because any of the operations aren't really expensive. 

Building a configuration parameters and different way of handling those lookups that you wan't to use for lookups again would be in my opionion more expensive and would add complexity where it isn't needed. Maybe I am missing something really obvious ..

Jon

Reply:
We already have all the metadata that should be required.